docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -267,385 +280,424 @@
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/EmailUI" target="_blank"><strong>Email UI</strong></a></p>
-			<p>This Email UI sample application demonstrates how to configure the screen using a variety of objects.<br />
+			<p>This sample application demonstrates how to configure the screen using a variety of objects.<br />
 			(This sample is only a reference for constructing an application's UI layout. Some menus and UI components do not provide real application's functions.)</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m6galleryui.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/GalleryUI" target="_blank"><strong>Gallery UI</strong></a></p>
-			<p>This Gallery UI demonstrates how to configure the screen using a variety of objects.<br />
+			<p>This sample application demonstrates how to configure the screen using a variety of objects.<br />
 			(This sample is intended to guide how to make you application UI but not functionality.)</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m7puzzle.png" width="150" /></td>
 			<td>
 			<p><a href="http://https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/Puzzle" target="_blank"><strong>Puzzle</strong></a></p>
-			<p>This is the basic UI sample application.</p>
+			<p>This sample application demonstrates the basic UI sample application.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m8snsui.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/SNSUI" target="_blank"><strong>SNS UI</strong></a></p>
-			<p>This SNS UI sample application demonstrates how to configure the screen using a variety of objects.<br />
-			It shows how to compose the view by using scroller, toolbar and drawer.<br />
+			<p>This sample application demonstrates how to configure the screen using a variety of objects.<br />
+			It shows how to compose the view by using scroller, toolbar, and drawer.<br />
 			(This sample is only a reference for constructing an application's UI layout. Some menus and UI components do not provide real application's functions.)</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m9settingsui.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/Settings" target="_blank"><strong>Settings UI</strong></a></p>
-			<p>This Setting UI sample application demonstrates how to configure the screen using a variety of objects.<br />
+			<p>This sample application demonstrates how to configure the screen using a variety of objects.<br />
 			(This sample is only a reference for constructing an application's UI layout. Some menus and UI components do not provide real application's functions.)</p>
```

Comment:
(This sample is only a reference for creating the UI layout of an application. Some menus and UI components do not provide the real functions of an application).